src/figuresClasses.ts

```diff
@@ -1,19 +1,84 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle',
+  color: 'red' | 'green' | 'blue',
```

Comment:
you need to identifier a separate type for this , and then use it in all places belove